picard/plugin3/manifest.py

```diff
@@ -19,9 +19,9 @@
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
 
 try:
-    from tomllib import load as load_toml
+    import tomllib
 except ImportError:
-    from tomlkit import load as load_toml
+    import tomli as tomllib
```

Comment:
We have more occurrences of those, did you check all?

```
picard/plugin3/api.py:268:                    import tomllib
picard/plugin3/registry.py:38:    import tomllib
scripts/extract_plugin_translations.py:193:        import tomllib
test/test_plugins3_helpers.py:119:        import tomllib
test/test_plugins3_migration.py:10:    import tomllib
```

Reply:
@zas yes, they all fall back to importing tomli. tomlkit was only used in `manifest.py`.